### PR TITLE
Update applications.md

### PR DESCRIPTION
**Pull request by kwfn**
adds entry for `delta`

### CODE REVIEW

File: src/applications.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Applications of Rust:
 * [_An Introduction to Chip-8 Emulation using the Rust Programming Language_](https://github.com/aquova/chip8-book/releases) - PDF book
 * [Book of crosvm](https://google.github.io/crosvm/) - a hosted (a.k.a. type-2) virtual machine monitor
+* [`delta`](https://dandavison.github.io/delta/introduction.html) - syntax-highlighting pager for git, diff, and grep output
```
Comment (glyn) [inline src/applications.md:6]: Why the back ticks?

**Review by glyn — APPROVED** [review]
I'm not sure the back ticks in the name of the book are necessary.

**Comment by kwfn** [comment]
I think because the name of the book is also the name of the command line interface.

**Comment by glyn** [comment]
> I think because the name of the book is also the name of the command line interface.

Ok, I can live with that, thanks.

**Pull request merged**